ga4gh/datamodel/rna_quantification.py

```
@@ -0,0 +1,390 @@
+    """
+    Defines an interface to a sqlite DB which stores all RNA quantifications
+    in the dataset.
+    """
+    def __init__(self, rnaQuantSqlFile="ga4gh-rnaQuant.db"):
+        super(SqliteRNABackend, self).__init__(rnaQuantSqlFile)
+        self.rnaQuantColumnNames = [f[0] for f in _rnaQuantColumns]
+        self.rnaQuantColumnTypes = [f[1] for f in _rnaQuantColumns]
+        self.expressionColumnNames = [f[0] for f in _expressionColumns]
+        self.expressionColumnTypes = [f[1] for f in _expressionColumns]
+
+    def searchRnaQuantificationsInDb(self, pageToken=0, pageSize=None,
+                                     rnaQuantificationId=None):
+        """
+        :param pageToken: int representing first record to return
+        :param pageSize: int representing number of records to return
+        :param rnaQuantificationId: string restrict search by id
+        :return an array of dictionaries, representing the returned data.
+        """
+        sql = ("SELECT * FROM RNAQUANTIFICATION")
+        sql_args = ()
+        if rnaQuantificationId is not None:
+            sql += " WHERE id = ? "
+            sql_args += (rnaQuantificationId,)
+        sql += sqliteBackend.limitsSql(pageToken, pageSize)
+        query = self._dbconn.execute(sql, sql_args)
+        try:
+            return sqliteBackend.sqliteRows2dicts(query.fetchall())
+        except AttributeError:
+            return None
+
+    def getRnaQuantificationById(self, rnaQuantificationId):
+        """
+        :param rnaQuantificationId: the RNA Quantification ID
+        :return: dictionary representing an RnaQuantification object,
+            or None if no match is found.
+        """
+        sql = ("SELECT * FROM RNAQUANTIFICATION WHERE id = ?")
+        query = self._dbconn.execute(sql, (rnaQuantificationId,))
+        try:
+            return sqliteBackend.sqliteRow2Dict(query.fetchone())
+        except AttributeError:
+            return None
```

Likewise here: I think it would be better to raise a custom exception here that would clearly identify the problem.